---
type: single
permalink: /agsx2025
---
<img src="/images/AGSx2025_Logo.png" width="100%">
<img src="/images/AGSx2025_QR_code.png" width="20%">

&nbsp;
<br>
# **<span style="color:purple"> AGSx Virtual Symposium Spring 2025 </span>**

__[Key Dates](#key_dates)__<br>
__[Overview](#overview)__<br>
__[Registration](#registration)__<br>
__[Schedule](#schedule)__<br>
__[Mailing List](#mailing_list)__<br>
__[Organizing Committee and Production Team](#organizers)__<br>

<a name="key_dates"></a> 
### **Key Dates**
ü¶ó [Tuesday, April 8](#apr8) - Session 1: Insect Genome Biology and Evolution <br>
ü™∞ [Tuesday, May 13](#may13) - Session 2: Genomic Advances in Non-Model Arthropod Pests <br>

<a name="overview"></a> 
### **Overview**
Welcome to the AGSx Virtual Symposium Spring 2025.  AGSx is organized and held independent of the annual Arthropod Genomics Symposium (AGS). AGSx symposia highlight recent advances in Arthropod genomics, empowered by genome assembly, editing and engineering, which are illuminating new perspectives in Arthropod taxonomy, evolution, physiology and management practices. This is an opportunity for additional exchange of ideas and community interaction. 

AGSx 2025 features two separate virtual webinars in April and May. Symposia sessions are listed below, and more specific details on speakers will be updated as information becomes available.  Questions and open discussions will follow speaker presentations.

Sessions will be recorded. Once available you will be able to click "Video Recording" at the top of the speaker information table for a link to the session's recording, available through the [i5k Community YouTube Channel](https://www.youtube.com/c/i5k_community). Subscribe to the channel to be notified when new videos are posted.

<a name="registration"></a> 
### **Registration**
Register for one or all symposia sections free of charge using [**this form**](https://forms.office.com/g/xcQtnzKC0g?origin=lprLink).  Additional information, including Zoom connection details for each session will be provided by email to those who have registered.

Additionally, those interested in continuing the discussions from these sessions or engaging more broadly with the i5k community are welcome to join the Arthropod Genomics Community Slack Workspace using [**this link**](https://join.slack.com/t/artgen/shared_invite/zt-1fut112rn-8c4cYs95eFytuCG84G3yvA).


<a name="schedule"></a> 
### **Schedule**
---
<a name="apr8"></a>
ü¶ó **<span style="color:purple"> April 8 </span>**
>Organized and moderated by Lindsey Perkin, USDA-ARS, [Insect Control and Cotton Disease Research Unit](https://www.ars.usda.gov/plains-area/college-station-tx/southern-plains-agricultural-research-center/insect-control-and-cotton-disease-research/), College Station, Texas USA

| __Session 1: Insect Genome Biology and Evolution__ | __[April 8, 2025 11am-1pm CST/12pm-2pm EST/6-8pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2025+Session+1&iso=20250408T12&p1=419&ah=2)__ |
| ---- | ----- |
| <img src="/images/AGSx2025_speaker_Song.png" width="200px"> __[Hojun Song](https://schistocerca.org/SongLab/)__ <br><br>Professor, Department of Entomology, Texas A&M University, College Station, Texas USA|<br>__Title:__ Evolution of Phenotypic Plasticity in Locusts As Inferred from Genomic Data<br><br>__Summary:__ The genus *Schistocerca* (Orthoptera: Acrididae) includes some of the most devastating locust species in the world, including the desert locust (*S. gregaria*), the Central American locust (*S. piceifrons*), and the South American locust (*S. cancellata*). These locust species show an extreme form of density-dependent phenotypic plasticity in which cryptic and shy individuals, known as the solitarious phase, can transform into conspicuous and gregarious individuals, known as the gregarious phase, in response to changes in local population density. In fact, this ‚Äúlocust phase polyphenism‚Äù is what makes the locusts distinctly different from regular grasshoppers. Intriguingly, *Schistocerca* includes 45 species, most of which are non-swarming sedentary grasshopper species, and phylogenetic studies have shown that the locust species do not form a monophyletic group, suggesting that locust phase polyphenism has evolved multiple times in the genus. Furthermore, recent experimental studies have indicated that some of the non-swarming grasshopper species show reduced density-dependent phenotypic plasticity, suggesting that *Schistocerca* as a whole is an exciting model clade that can be used to study how phenotypic plasticity has evolved as species diverge.<br><br>Until now, it has been very challenging to study these insects from a genomic perspective because of their large genome sizes (up to 9.1 Gb), which are the largest among insects. Recently, however, we launched the Behavioral Plasticity Research Institute (BPRI) supported by the National Science Foundation Biology Integration Institute (NSF-BII) program, which allowed us to produce high-quality chromosome-length assemblies of six *Schistocerca* species, including three locust and three non-swarming grasshopper species. These genomes were generated using PacBio HiFi and Hi-C technologies with a target coverage of 30x and 15x, respectively. Our assemblies are highly contiguous (contig N50 ranging from 27.4 to 74.2 Mb and scaffold N50 ranging from 791.2 to 854.0 Mb) and consistent with known estimates of chromosome count (11 autosomal pairs + sex chromosome) for these species. In this presentation, we introduce these new genomes as well as other omics approaches that we are currently pursuing, including transcriptomics, epigenomics, and single-cell genomics. We also introduce the overall vision and research activities of the BPRI, and make a strong case for why *Schistocerca* is an excellent model clade for biological integration.<br><br>Websites: schistocerca.org/SongLab; behavioralplasticity.org <br><br>|
| <img src="/images/AGSx2025_speaker_Walt.png" width="200px"> __[Hunter K. Walt](https://www.researchgate.net/profile/Hunter-Walt)__ <br><br>Postdoctoral Associate, Department of Biochemistry, Nutrition, and Health Promotion, Mississippi State University, Starkville, Mississippi USA|<br>__Title:__ Genomic Analysis Sheds Light on a Stink Bug's High Damage to Soybean<br><br>__Summary:__ The redbanded stink bug (Hemiptera: Pentatomidae) is a soybean pest found from South America up through the Southeastern United States. Previous studies observed that the redbanded stink bug causes more physical damage to soybean than other stink bugs, likely due to an enzymatic component of its saliva. However, the genomic basis of this trait remains unexplored. To address this, we used a publicly available chromosome-level genome assembly and transcriptomic data from the salivary tissues of the redbanded stink bug to investigate its high damage to soybean. We hypothesized that (1) lineage-specific expansions of digestion-related gene families occurred in the redbanded stink bug genome, and (2) these expanded gene families are differentially expressed in the salivary tissues. Contrary to our hypothesis, we did not find an overrepresentation of duplicated genes in the set of differentially expressed salivary genes. Instead, we found that differential retention of proteolytic genes more accurately explains the redbanded stink bug‚Äôs high damage to soybean. Additionally, we found multiple differentially expressed copies of a glycoside hydrolase gene that was acquired via horizontal gene transfer from bacteria in the redbanded stink bug salivary gland. This gene family is widely present in phytophagous stink bugs and seed bugs and likely plays a role and plant tissue digestion. <br><br>|
| <img src="/images/AGSx2025_speaker_Fritz.png" width="200px"> __[Megan Fritz](https://www.meganfritzlab.com/)__ <br><br>University of Maryland, College Park, Maryland USA|<br>__Title:__ Genome evolution in a North American lepidopteran species in response to modern agricultural practices<br><br>__Summary:__ The evolution of insect resistance to agricultural management practices is a long-standing problem in food production systems, which results in crop failure and economic losses. Historically, most cases of field-evolved insect resistance have been attributed to *de novo* mutations of large effect at a single locus. As a result, many of the agricultural policies and practices for insect resistance management have been shaped under assumptions of single locus resistance evolution. The phytophagous North American insect pest, *Helicoverpa zea*, has recently evolved high levels of resistance to corn and cotton expressing transgenes from the bacterium *Bacillus thuringiensis* (Bt). Our work has examined the genome architecture of field-evolved Bt resistance in *H. zea*, using whole genome sequencing, differential gene expression analysis, and quantitative genetic assays. We show that adaptation to Bt crops by *H. zea* does not result from *de novo* mutation at a single locus but is the result of multi-locus selection on standing genetic variation across the genome. Our results demonstrate the complex nature of resistance evolution in pests of agricultural ecosystems. <br><br>|


---
<a name="may13"></a>
ü™∞ **<span style="color:purple"> May 13 </span>**
>Organized and moderated by Perot Saelao and David Luecke, USDA-ARS, Veterinary Pest Genetics Research Unit, Kerrville, Texas USA

| __Session 2: Genomic Advances in Non-Model Arthropod Pests__ | __[May 13, 2024 11am-1pm CST/12pm-2pm EST/6-8pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2025+Session+2&iso=20250513T12&p1=419&ah=2)__ |
| ---- | ----- |
| <img src="/images/AGSx2025_speaker_Johnson.png" width="200px"> __[Michelle Johnson](https://www.linkedin.com/in/michelle-johnson-1871a0154)__ <br><br>Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, California USA|<br>__Title:__ *TBD*<br><br>__Summary:__  <br><br>|
| <img src="/images/AGSx2025_speaker_Bista.png" width="200px"> __[Basanta Bista](https://www.linkedin.com/in/basanta-bista-485077135/)__ <br><br>Department of Biology and Biochemistry, University of Houston, Houston, Texas USA|<br>__Title:__ Genomic Landscape of *Musca domestica*<br><br>__Summary:__  <br><br>|
| <img src="/images/AGSx2025_speaker_Kneubehl.png" width="200px"> __[Alexander Kneubehl](https://www.bcm.edu/people-search/alexander-kneubehl-24579)__ <br><br>Baylor College of Medicine, Houston, Texas USA|<br>__Title:__ *TBD*<br><br>__Summary:__  <br><br>|
| <img src="/images/AGSx2025_speaker_Lopez.png" width="200px"> __[Job Lopez](https://www.bcm.edu/people-search/job-lopez-25566)__ <br><br>Baylor College of Medicine, Houston, Texas USA|<br>__Title:__ A functional genomics approach to delineate the molecular mechanisms of argasid oogenesis<br><br>__Summary:__  <br><br>|


<a name="mailing_list"></a> 
### **Mailing List**
To keep up to date with news about this conferece, the i5k webinar series and other i5k activities, please subscribe to the Arthropod News [mailing list](https://listserv.ksu.edu/?SUBED1=ARTHROPODNEWS&A=1). Email frequency is typically very low.

<a name="organizers"></a> 
### **Organizing Committee and Production Team**
This virtual symposium series was brought to you through the efforts of the following organizing committee and production team:
- Brad Coates, USDA-ARS, Ames, Iowa USA
- Perot Saelao, USDA-ARS, Kerrville, Texas USA
- David Luecke, USDA-ARS, Kerrville, Texas USA
- Lindsey Perkin, USDA-ARS, College Station, Texas USA
- Pia Olafson, USDA-ARS, Kerrville, Texas USA
- Glenn Hanes, USDA-ARS, Beltsville, Maryland USA
- Anna Childers, USDA-ARS, Beltsville, Maryland USA
- Brenda Oppert, USDA-ARS, Manhattan, Kansas USA
- [Robert Waterhouse](https://rmwaterhouse.org), SIB Swiss Institute of Bioinformatics, Switzerland
