---
type: single
permalink: /agsx2024
---
<img src="/images/AGSx2024_AGR_i5K_Beenome100_CGRv1_crop.png" width="100%">
<img src="/images/AGSx2024_QR_code.png" width="20%">

&nbsp;
<br>
# **<span style="color:purple"> AGSx Virtual Symposium Spring 2024 </span>**

__[Key Dates](#key_dates)__<br>
__[Overview](#overview)__<br>
__[Registration](#registration)__<br>
__[Schedule](#schedule)__<br>
__[Mailing List](#mailing_list)__<br>
__[Organizing Committee and Production Team](#organizers)__<br>

<a name="key_dates"></a> 
### **Key Dates**
ðŸ¦—: [Tuesday, February 13](#feb13) - Session 1: Production of Insects for Food and Feed <br>
:beetle: [Wednesday, April 10](#apr10) - Session 2: Insect Genome Biology and Evolution <br>
:bee: [Tuesday, April 30](#apr30) - Session 3: Beenome100 and Comparative Bee Genomics <br>
:butterfly: [Tuesday, May 21](#may21) - Session 4: NIH Comparative Genomics Resource: An NCBI Toolkit of Data and Tools Unlocking Arthropod Research <br>

<a name="overview"></a> 
### **Overview**
Welcome to the AGSx Virtual Symposium Spring 2024.  AGSx is organized and held independent of the annual Arthropod Genomics Symposium (AGS).  Goals of the AGSx symposia are to highlight recent advances in Arthropod genomics, on topics from genome assembly, editing and engineering, and focused areas of insects as food and honey bee research.  This is an opportunity for additional exchange of ideas and community interaction. 

AGSx 2024 is composed of four separate virtual webinars occuring between February and May. Symposia sessions are listed below, and more specific details on speakers will be updated as information becomes available.  Questions and open discussions will follow speaker presentations.

Sessions will be recorded. Once available you will be able to click "Video Recording" at the top of the speaker information table for a link to the session's recording, available through the [i5k Community YouTube Channel](https://www.youtube.com/c/i5k_community). Subscribe to the channel to be notified when new videos are posted.

<a name="registration"></a> 
### **Registration**
Participants are directed to register for one or all symposia sections free of charge using [**this form**](https://forms.office.com/pages/responsepage.aspx?id=5zZb7e4BvE6GfuA8-g1Gl4GIwkebKtxBijDWWCNgqZ5UQjNJM0tXVDE3SEE2WDZQS0VEUk9JUkNFNS4u&origin=lprLink).  Additional information, including Zoom connection details for each session will be provided by email to those who have registered.

Additionally, those interested in continuing the discussions from these sessions or engaging more broadly with the i5k community are welcome to join the Arthropod Genomics Community Slack Workspace using [**this link**](https://join.slack.com/t/artgen/shared_invite/zt-1fut112rn-8c4cYs95eFytuCG84G3yvA).


<a name="schedule"></a> 
### **Schedule**

<a name="feb13"></a>
ðŸ¦—: **<span style="color:purple"> February 13 </span>** 
>Organized and moderated by [Kristin Duffield](https://www.ars.usda.gov/midwest-area/peoria-il/national-center-for-agricultural-utilization-research/cbp/people/kristin-duffield/), USDA-ARS, [National Center for Agricultural Utilization Research](https://www.ars.usda.gov/midwest-area/peoria-il/national-center-for-agricultural-utilization-research/), Peoria, IL USA & [Brenda Oppert](https://www.ars.usda.gov/plains-area/mhk/cgahr/spieru/people/brenda-oppert/), USDA ARS, [Stored Product Insect and Engineering Research Unit](https://www.ars.usda.gov/plains-area/mhk/cgahr/spieru/), Manhattan, KS USA

| __Session 1: Genomics for Insects as Food and Feed__ | __[February 13, 2024 10am-12pm CST/11am-1pm EST/5-7pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2024+Session+1&iso=20240213T11&p1=419&ah=2)__ |
| ---- | ----- |
| ![Monica Pava-Ripoll](/images/AGSx2024_speaker_PavaRipoll.png) __[Monica Pava-Ripoll](https://www.linkedin.com/in/monica-pava-ripoll-ph-d-33b25887)__ <br><br>Research Entomologist, U.S. FDA, Center for Food Safety and Applied Nutrition (CFSAN), Office of Food Safety (OFS), College Park, Maryland USA |<br>__Title:__ A Metagenomic Next-Generation Sequencing approach for evaluating the identity of insects as food<br><br>__Summary:__ In markets all over the world, a wide variety of insect species are offered for sale as food or feed. While some insects are farmed, others are harvested in the wild from forests or agricultural fields. As of right now, there are no established laboratory procedures to verify the identity of insect products sold as food or feed. The use of a Metagenomic Next-Generation Sequencing (NGS) method to identify whole insects and products derived from insects that are marketed as food or food ingredients will be the main topic of this presentation. The safety concerns related to insects as food will also be briefly covered in this presentation. <br><br>|
| ![Carlotta Savio](/images/AGSx2024_speaker_Savio.png) __[Carlotta Savio](https://www.linkedin.com/in/carlotta-savio-48930a140)__ <br><br>Research Fellow, University of Turin, Piedmont, Italy |<br>__Title:__ Unravelling the role of potential bacterial probiotics for preserving insect health<br><br>__Summary:__ Interactions between insects and their microbiota affect insect behaviour and evolution. When specific microorganisms are provided as a dietary supplement, insect reproduction, food conversion and growth are enhanced and health is improved in cases of nutritional deficiency or pathogen infection. Insectâ€“microbiota interactions and the presence of beneficial microbes play an essential role in preserving insect health. Probiotic provision to insects reared for food and feed purposes could decrease the incidence of environmental and biological stress in mass-rearing conditions. A workflow for selecting bacterial strains for insect species reared for food and feed is proposed along with methods used to isolate and measure the effectiveness of a probiotic. A last discussion is focused on future perspectives on probiotic applications in mass-reared insects and the role of microbiota in preserving insect health. <br><br>|
| ![Elida Espinoza](/images/AGSx2024_speaker_Espinoza.png) __[Elida Espinoza](https://www.linkedin.com/in/elidaespinoza/)__ <br><br>Research Entomologist (Genetics), Genetics R&D at [EnviroFlight](https://www.enviroflight.net/), Apex, North Carolina USA|<br>__Title:__ Using biomarkers in black soldier fly breeding<br><br>__Summary:__ The rearing of farmed insects is becoming a vital component of food and feed production systems seeking to establish a positive impact on health, welfare, and the environment. Advances in breeding can accelerate the establishment of the quality traits that will drive the success of farmed insect populations, this includes the development of tools to measure and record relevant performance data. For example, biomarkers can be developed to measure the presence, progression or predisposition of particular conditions to gain valuable insights in the overall health of the insects reared for animal feed. This presentation will discuss potential biomarkers for studying various conditions important in the commercial farming of the black soldier fly (BSF) (*Hermetia illucens*). <br><br>|


---
<a name="apr10"></a>
:beetle: **<span style="color:purple"> April 10 </span>**
>Organized and moderated by [Lindsey Perkin](https://www.ars.usda.gov/people-locations/person?person-id=53466), USDA-ARS, [Insect Control and Cotton Disease Research Unit](https://www.ars.usda.gov/plains-area/college-station-tx/southern-plains-agricultural-research-center/insect-control-and-cotton-disease-research/), College Station, TX, USA

| __Session 2: Insect Genome Biology and Evolution__ | __[April 10, 2024 10am-12pm CST/11am-1pm EST/5-7pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2024+Session+2&iso=20240410T11&p1=419&ah=2)__ |
| ---- | ----- |
| ![Kirsten Verster](/images/AGSx2024_speaker_Verster.png) __[Kirsten Verster](https://hadlylab.stanford.edu/people/kirsten-verster)__ <br><br>Postdoctoral Scholar, Stanford University, Stanford, California USA|<br>__Title:__ A toxic triangle: how insects co-opted phage genes for defense against deadly wasps<br><br>__Summary:__ Several disease-causing bacteria produce toxins that damage host cells by triggering preprogrammed cell death. Two such bacterial toxins are called cytolethal distending toxin B and apoptosis-inducing protein of 56 kDa. We discovered that diverse insect species coopted the two bacterial genes encoding each cytotoxin through a phenomenon called horizontal gene transfer (HGT). HGT occurs when a gene from one organism is inserted into the genome of another and then is stably inherited across generations. We found that these two bacterial toxin genes were captured by species in the *Drosophila ananassae* subgroup, which are highly resistant to parasitoid wasps, and we observe that *D. ananassae* lines carrying null mutations of these genes are more susceptible to parasitoid infection than the wild type. We conclude that toxin cargo genes were captured by these insects millions of years ago and integrated as novel anti-parasitoid modules in their innate immune system. <br><br>|
| ![Ryan Bracewell](/images/AGSx2024_speaker_Bracewell.png) __[Ryan Bracewell](https://biology.indiana.edu/about/faculty/bracewell-ryan.html)__ <br><br>Assistant Professor, Department of Biology, University of Indiana, Bloomington, Indiana USA|<br>__Title:__ Using genomics to explore the sex chromosomes of beetles<br><br>__Summary:__ Beetles are an incredibly species-rich group of animals. Most species are thought to have XY sex chromosomes although variation exists within the group. Until recently, it was unclear if the X chromosome was conserved over evolutionary time in beetles and if the X might have taken on unique properties that set it apart from the autosomes. In this talk, I will detail comparative analyses looking at sex chromosome evolution in beetles and will discuss work we have been doing exploring the tempo of sex chromosome differentiation after sex chromosomes fuse to autosomes. These so-called neo-sex chromosomes provide a unique opportunity to study how X and Y chromosomes differentiate from one another and evolve changes in gene content, gene expression, and transposable element abundance. These genomic changes could potentially lay the foundation for the evolution of reproductive isolation between populations, thereby allowing new species to form. <br><br>|
| ![Michelle Jonika](/images/AGSx2024_speaker_Jonika.png) __[Michelle Jonika](https://michellejonika.github.io/)__ <br><br>Postdoctoral Research Scholar, [Blackmon Lab](https://coleoguy.github.io/), [Department of Biology](https://www.bio.tamu.edu/), Texas A&M University, College Station, Texas USA|<br>__Title:__ The coevolution of binary traits and genome structure in insects<br><br>__Summary:__ Insect taxonomistsâ€™ early adoption of cytogenetics has led to an incredible amount of data describing genome structure across insects. Chromosome number is a fundamental genomic trait that is often the first recorded characteristic of a genome. We observe striking heterogeneity in the range of chromosome numbers among clades across the tree of life. This tells us that there must be great heterogeneity in rates of chromosome number evolution among lineages. Despite recent developments in comparative methods, most of this heterogeneity is still poorly understood. Our lab has developed a comparative method that allows for a binary trait to coevolve with and impact rates of chromosome number evolution. This comparative method allows us to identify coevolving traits that lead to heterogeneity in rates of chromosome number across the tree of life. For example, this method has allowed us to look at the fitness effect of mutations that change chromosome number in beetles as well as the impact of segregation mechanisms on chromosome number evolution across all insects. Additionally, running a simpler model of chromosome number evolution without a coevolving binary trait shows us that insect orders vary dramatically in the overall rate of chromosome number evolution, a proxy of genome structural stability, and the pattern of evolution. These findings have important implications for our understanding of likely modes of speciation and offer insight into the most informative clades for future genome sequencing. <br><br>|


---
<a name="apr30"></a>
:bee: **<span style="color:purple"> April 30 </span>**
>Organized and moderated by [Jay Evans](https://www.ars.usda.gov/people-locations/person?person-id=10065), USDA-ARS, [Bee Research Laboratory](https://www.ars.usda.gov/northeast-area/beltsville-md-barc/beltsville-agricultural-research-center/bee-research-laboratory/), Beltsville, MD, USA & [Michael Branstetter](https://www.ars.usda.gov/people-locations/person/?person-id=52530), USDA-ARS, [Pollinating Insects â€“ Biology, Management and Systematics Research Unit](https://www.ars.usda.gov/pacific-west-area/logan-ut/pollinating-insect-biology-management-systematics-research/) Logan, UT, USA

| __Session 3: Beenome100 and Comparative Bee Genomics__ | __[April 30, 2024 10am-12pm am CST/ 11am-1pm EST/ 5-7pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2024+Session+3&iso=20240430T11&p1=419&ah=2)__ |
| ---- | ----- |
| ![Michael Branstetter](/images/AGSx2024_speaker_Branstetter.png) <br> __[Michael Branstetter](https://www.ars.usda.gov/people-locations/person/?person-id=52530)__ <br><br>USDA-ARS, [Pollinating Insects â€“ Biology, Management and Systematics Research Unit](https://www.ars.usda.gov/pacific-west-area/logan-ut/pollinating-insect-biology-management-systematics-research/) Logan, Utah USA|<br>__Title:__ Beenome100: Bee diversity through a genomic lens<br><br>__Summary:__ The Beenome100 project is a collaborative initiative led by USDA that aims to generate 100+ new, reference quality genomes for bees utilizing the latest sequencing and analytical approaches. Although improving honey-bee genomics is a goal, the project emphasizes the sampling of taxonomically diverse native and introduced bees in the United States. In this talk I will provide an overview of the Beenome100 project, covering project aims, collaborators, taxonomic prioritization, sample collection, sequencing, and data processing. I will then provide an update about current progress and present some results, highlighting improved bee genomes for species of agricultural importance and how the improved taxonomic coverage will enable genomic comparisons across the bee tree of life in much greater detail than ever before. <br><br>|
| ![Rena Schweizer](/images/AGSx2024_speaker_Schweizer.png) <br> __[Rena Schweizer](https://renaschweizer.org/)__ <br><br>USDA-ARS, [Pollinating Insects â€“ Biology, Management and Systematics Research Unit](https://www.ars.usda.gov/pacific-west-area/logan-ut/pollinating-insect-biology-management-systematics-research/) Logan, Utah USA|<br>__Title:__ The Beenome100 Project: Enabling bee conservation in the 'omics era<br><br>__Summary:__ Wild bees are critical to the health and productivity of the worldâ€™s natural and agricultural systems, yet many bees are declining due to disturbance, pathogens, habitat loss, and climate change. Genomics enables powerful insights into key population parameters that are critical for assessing and managing species of conservation concern. The Beenome100 project was initiated to sequence and assemble high-quality reference genomes for over 100 species of bees, and to provide genomic data for a number of threatened and/or endangered species of conservation and management concern. In this talk, I will present our recent progress on generating reference genomes for endangered and/or threatened species such as the Mojave poppy bee, the western bumble bee, Franklinâ€™s bumble bee, the rusty-patched bumble bee, and others. I will highlight adjacent genome resequencing projects aimed at understanding population genetics of threatened species such as Franklinâ€™s bumble bee. Our results highlight the utility of genomics in conservation research and present new insights into the decline of native bee species. <br><br>|
| ![Karen Kapheim](/images/AGSx2024_speaker_Kapheim.png) <br> __[Karen Kapheim](https://www.kapheimlab.com)__ <br><br>Utah State University, Logan, Utah USA <br><br> and <br><br> ![Sarah Kocher](/images/AGSx2024_speaker_Kocher.png) <br> __[Sarah Kocher](https://kocherlab.princeton.edu)__ <br><br>Princeton University, Princeton, New Jersey USA|<br>__Title:__ Harnessing natural variation to study the evolution of social behavior<br><br>__Summary:__ Natural variation can provide powerful insights into the genetic and environmental factors shaping the origins and evolution of complex traits. Bees harbor extraordinary variation in social behavior within and between species. Throughout their evolutionary history, bees have repeatedly gained and lost sociality, resulting in many closely related yet behaviorally variable species. Building on the wealth of natural history data for this system, we have developed complementary genomic and experimental resources to identify the molecular and physiological mechanisms underlying variation in social behavior. We have discovered changes in hormone signaling, developmental pathways, sensory systems, and neural processes that are tightly associated with the gains and losses of sociality. We have also taken a theoretical approach to identify several key environmental factors that shape social evolution. Taken together, this work broadens our understanding of social behavior across scales, from the molecular building blocks of the â€˜social brainâ€™ to the ecological and evolutionary forces that drive social evolution. <br><br>|
| ![Katie Dogantzis](/images/AGSx2024_speaker_Dogantzis.png) <br> __[Katie Dogantzis](https://www.kathleendogantzis.com/)__ <br><br>York University, Toronto, Ontario Canada|<br>__Title:__ Honey bee population genomics<br><br>__Summary:__ The honey bee, *Apis mellifera*, is widely distributed across its native range in Africa, Europe, and parts of Asia. This broad distribution has led to the evolution of several distinct subspecies and lineages, characterized by variations in morphology, geography, and genetics. Exploring the genetic foundation of this species diversification is essential for disentangling the species origins and understanding how the honey bee genome diverged to facilitate adaptation. In this presentation, I will discuss how we used an extensive population genomic dataset comprising 251 honey bee genomes representing 18 putative subspecies to shed light on the evolutionary and adaptive history of *Apis mellifera*. These results highlight the influence of lineage specific mutations, repeated selection, and worker-related traits. <br><br>|


---
<a name="may21"></a>
:butterfly: **<span style="color:purple"> May 21 </span>**
>Organized and moderated by Terence Murphy, National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, MD, USA

| __Session 4: NIH Comparative Genomics Resource: An NCBI Toolkit of Data and Tools Unlocking Arthropod Research__ | __[May 21, 2024 10am-12pm CST/11am-1pm EST/5-7pm CET](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSx+2024+Session+4&iso=20240521T11&p1=419&ah=2)__ |
| ---- | ----- |
| ![Terence D. Murphy](/images/AGSx2024_speaker_Murphy.jpg) <br> __Terence D. Murphy__ <br><br>National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, Maryland USA|<br>__Title:__ The NIH Comparative Genomics Resource (CGR): Organism-agnostic support for reliable eukaryotic comparative genomics<br><br>__Summary:__ Comparative genomic analyses are a powerful tool for scientific discovery. The National Institutes of Health (NIH) Comparative Genomics Resource (CGR) maximizes the impact of eukaryotic research organisms and their genomic data to biomedical research. CGR facilitates reliable comparative genomics analyses for all eukaryotic organisms through community collaboration and a National Center for Biotechnology Information (NCBI) genomics toolkit. The toolkit includes high-quality data, tools, and interfaces for connecting community-provided resources with NCBI. This presentation will give an overview of CGR goals and the opportunities within the arthropod genomics space.<br><br>|
| ![Vamsi K. Kodali](/images/AGSx2024_speaker_Kodali.jpg) <br> __Vamsi K. Kodali__ <br><br>National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, Maryland USA|<br>__Title:__ Genome annotation at NCBI<br><br>__Summary:__ Genome assemblies submitted to the INSDC databases (GenBank, ENA and DDBJ) are processed by RefSeq at NCBI and annotated using the Eukarytoic Genome Annotation Pipeline (EGAP). This presentation will cover the journey a genome takes to become a RefSeq assembly. I will focus on the contamination screening process as well as the structural and functional annotation carried out through EGAP. Additionally, I will introduce EGAPx â€“ an early release version of EGAP designed for annotating genomes by users outside of NCBI environment.<br><br>|
| ![Eric Cox](/images/AGSx2024_speaker_Cox.jpg) <br> __Eric Cox__ <br><br> along with <br><br> __Nuala O'Leary and Mirian T.N. Tsuchiya__ <br><br>National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, Maryland USA|<br>__Title:__ NCBI Datasets: Simplifying Access to Insect Genes and Genomes<br><br>__Summary:__ NCBI Datasets is a resource from the National Center for Biotechnology Information (NCBI) that provides easy and intuitive access to sequence and metadata for a broad range of taxa, including insects. <br>This presentation will highlight recent updates to the Datasets website with examples of insect gene, ortholog and genome data retrieval. We will also present our new taxonomy details page with options to download taxonomy data and we will introduce our command-line tools.<br><br>|
| ![Dong-Ha Oh](/images/AGSx2024_speaker_Oh.jpg) <br> __Dong-Ha Oh__ <br><br>National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, Maryland USA|<br>__Title:__ Visualizing insect genome alignments in NCBI Comparative Genome Viewer (CGV)<br><br>__Summary:__ The NCBI Comparative Genome Viewer (CGV) offers interactive browsing of pairwise genome alignments and visualization of macro- and micro-synteny between chromosomes and orthologous genomic regions. Alignment blocks in CGV are linked to the Genome Data Viewer (GDV), where users can explore alignments in nucleotide resolution, along with gene models and other genomic data tracks, including user-supplied ones. Currently, CGV hosts alignments for more than 80 pairs of insect genomes. Users can request any pair of NCBI genomes to be added to CGV if both genomes are assembled as chromosomes and generally in the same order.<br><br>|
| ![Terence D. Murphy](/images/AGSx2024_speaker_Murphy.jpg) <br> __Terence Murphy__ <br><br>National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), National Institutes of Health (NIH), Bethesda, Maryland USA|<br>__Concluding remarks:__ With CGR, NCBI aims to serve as a central hub for accessing data and connecting to community resources to drive the next wave of comparative genomics research. To succeed, CGR is highly dependent on community feedback, and we encourage feedback via email through the contact information on the [CGR website](https://www.ncbi.nlm.nih.gov/cgr) to help direct our future efforts. This work was supported by the National Center for Biotechnology Information of the National Library of Medicine (NLM), National Institutes of Health.<br><br>|
| |<br>__Q&A and Live Demo Session__ <br><br>|


<a name="mailing_list"></a> 
### **Mailing List**
To keep up to date with news about this conferece, the i5k webinar series and other i5k activities, please subscribe to the Arthropod News [mailing list](https://listserv.ksu.edu/?SUBED1=ARTHROPODNEWS&A=1). Email frequency is typically very low.

<a name="organizers"></a> 
### **Organizing Committee and Production Team**
This virtual symposium series was brought to you through the efforts of the following organizing committee and production team:
- [Kristin Duffield](https://www.ars.usda.gov/midwest-area/peoria-il/national-center-for-agricultural-utilization-research/cbp/people/kristin-duffield/), USDA-ARS, Peoria, IL USA
- [Brenda Oppert](https://www.ars.usda.gov/plains-area/mhk/cgahr/spieru/people/brenda-oppert/), USDA-ARS, Manhattan, KS, USA
- [Lindsey Perkin](https://www.ars.usda.gov/people-locations/person?person-id=53466), USDA-ARS, College Station, TX, USA
- [Jay Evans](https://www.ars.usda.gov/people-locations/person?person-id=10065), USDA-ARS, Beltsville, MD, USA
- [Michael Branstetter](https://www.ars.usda.gov/people-locations/person/?person-id=52530), USDA-ARS, Logan, UT, USA
- Terence Murphy, National Center for Biotechnology Information (NCBI), Bethesda, MD, USA
- [Pia Olafson](https://www.ars.usda.gov/people-locations/person/?person-id=33760), USDA-ARS, Kerrville, TX, USA
- [Glenn Hanes](https://www.ars.usda.gov/people-locations/person?person-id=2280), USDA-ARS, Beltsville, MD, USA
- [Anna Childers](https://www.ars.usda.gov/people-locations/person?person-id=51956), USDA-ARS, Beltsville, MD, USA
- [Robert Waterhouse](https://rmwaterhouse.org), SIB Swiss Institute of Bioinformatics, Switzerland
- [Brad Coates](https://www.ars.usda.gov/people-locations/person?person-id=1047), USDA-ARS, Ames, IA, USA
