---
type: single
permalink: /agsx2021
---
<img src="/images/AGSX_Banner_Event.png" width="100%">
<img src="/images/AGSX_Banner_Sponsors.png" width="100%"> 
&nbsp;
<br>
# **<span style="color:purple"> AGSX Virtual Symposium Spring 2021 </span>**

__[Key Dates](#key_dates)__<br>
__[Overview](#overview)__<br>
__[Registration](#registration)__<br>
__[Schedule](#schedule)__<br>
__[Mailing List](#mailing_list)__<br>
__[Organizing Committee and Production Team](#organizers)__<br>

<a name="key_dates"></a> 
### **Key Dates**
[Tuesday, February 9](#feb9) - Session 1: Insect Genomic Technologies to Improve Food Applications <br/>
[Tuesday, March 9](#mar9) - Session 2: Arthropod Genomics and Genome Engineering <br/>
[Tuesday, April 13](#apr13) - Session 3: Application of New Genomic Tools and Techniques in Arthropods <br/>
[Wednesday, May 12](#may12) - Session 4: Honeybee Workshop <br/>

<a name="overview"></a> 
### **Overview**
Welcome to the AGSX Virtual Symposium Spring 2021.  AGSX is organized and held independent of the annual Arthropod Genomics Symposium (AGS).  Goals of the AGSX symposia are to highlight recent advances in Arthropod genomics, on topics from genome assembly, editing and engineering, and focused areas of insects as food and honey bee research.  This is an opportunity for additional exchange of ideas and community interaction. 

AGSX is composed of four separate virtual webinars from February to May, 2021. Symposia sessions are listed below, and more specific details on speakers will be updated as information becomes available.  Questions and open discussions will follow speaker presentations.

Sessions are being recorded and videos are posted within a week of when the session occurs. Click "Vido Recording" at the top of the speaker information table for a link to the session's recording, available through the i5k Community YouTube Channel. Subscribe to the channel to be notified when new videos are posted.

<a name="registration"></a> 
### **Registration**
Participants are directed to register for one or all symposia sections free of charge using [**this form**](https://tinyurl.com/yyoqruyb).  Additional information on presentations and Zoom connections for each session will be announced via a registration email list.

Additionally, those interested in continuing the discussions from these sessions or engaging more broadly with the i5k community are welcome to join the Arthropod Genomics Community Slack Workspace using [**this link**](https://join.slack.com/t/artgen/shared_invite/zt-m1vevfbp-kq_0uXl5q8FGO1iiSDaZFw).


<a name="schedule"></a> 
### **Schedule**

<a name="feb9"></a>
**<span style="color:purple"> February 9 </span>** <br/>
Organized and moderated by Brenda Oppert, USDA ARS, Stored Product Insect and Engineering Research Unit, Manhattan, KS.


| __Session 1: Insect Genomic Technologies to Improve Food Applications__ | __[Video Recording](https://youtu.be/5NWpc-06Qfg)__ |
| ---- | ----- |
| __[Chris Jiggins](https://heliconius.zoo.cam.ac.uk/people/chris-jiggins/)__ <br><br>Professor of Evolutionary Biology, Department of Zoology, University of Cambridge, Cambridge, UK |<br>__Title:__ A high-quality, chromosome-level genome assembly of the Black Soldier Fly (*Hermetia Illucens* L.) and its use in characterising global genetic diversity<br><br>__Summary:__ *Hermetia illucens* L. (Diptera: Stratiomyidae), the Black Soldier Fly (BSF) is an increasingly important mass reared resource for bioconversion of organic material into animal feed. We generated a high-quality chromosome-scale genome assembly of the BSF using Pacific Bioscience, 10X Genomics linked read and high-throughput chromosome conformation capture sequencing technology. Scaffolding the final assembly with Hi-C data produced a highly contiguous 1.01 Gb genome with 99.75% of scaffolds assembled into pseudo-chromosomes representing seven chromosomes with 16.01 Mb contig and 180.46 Mb scaffold N50 values. The highly complete genome obtained a BUSCO completeness of 98.6%. We masked 67.32% of the genome as repetitive sequences and annotated a total of 17,664 protein-coding genes using the BRAKER2 pipeline. We analyzed an established lab population to investigate the genomic variation and architecture of the BSF revealing six autosomes and the identification of an X chromosome. We have been using this genome to conduct whole genome resequencing of strains from around the world, showing marked divergence between captive strains and wild strains that are more closely related to South American wild strains. Release of this novel chromosome-scale BSF genome assembly will provide an improved platform for further genomic studies and functional characterization of candidate regions of artificial selection.<br><br>|
| <img src="/images/AGSX_speaker_Kristin Duffield.png"/> __[Kristin Duffield](https://www.ars.usda.gov/midwest-area/peoria-il/national-center-for-agricultural-utilization-research/cbp/people/kristin-duffield/)__ <br><br>Postdoctoral Research Entomologist, USDA ARS, Crop Bioprotection Research Unit, NCAUR, Peoria, IL, USA |<br>__Title:__ Catching the bug: A path towards understanding the diseases of reared crickets<br><br>__Summary:__ We will be tasked with almost doubling food production by 2050; thus, interest in developing sustainable protein alternatives has gained incredible momentum. One such alternative is insect-derived protein for both direct and indirect (via animal feed) human consumption. Crickets (family: Gryllidae) are an especially popular group of edible insects due to their nutritional quality and palatability. However, we know little about the diseases that are impacting the health of this emerging insect crop. Here, I identified, characterized, and quantified a pathogenic virus isolated from a colony of lab-reared decorated crickets (Gryllodes sigillatus) using various molecular and genomic techniques. I will discuss these results and highlight the future considerations necessary for ensuring the health of farmed insects.<br><br>|


<a name="mar9"></a>
**<span style="color:purple"> March 9 </span>** <br/>
Organized and moderated by Dr. Lindsey Perkin, USDA-ARS, College Station, TX, USA


| __Session 2: Arthropod Genomics and Genome Engineering__ | __[Video Recording](https://youtu.be/M9ZmWaGqq-8)__ |
| ---- | ----- |
| <img src="/images/AGSX_speaker_Amanda_Stahlke.png"/> __[Amanda Stahlke](https://amandastahlke.weebly.com/)__ <br><br>University of Idaho, Moscow, ID |<br>__Title:__ Improving our understanding of rapid evolution with genomic resources in a hybridizing, range expanding biocontrol agent, the tamarisk beetle (*Diorhabda* spp.)<br><br>__Summary:__ As planned invasions, classical biological control (biocontrol) agents present important opportunities to better understand the eco-evolutionary mechanisms of establishment, spread, and rapid evolution in a novel environment. Despite this potential, genomic tools have rarely been used in biocontrol agents of invasive plants. The case of four tamarisk beetle species, *Diorhabda carinata*, *D. carinulata*, *D. elongata*, and *D. sublineata* (Coleoptera: Chrysomelidae), biocontrol agents introduced to control the invasive woody shrub Tamarix in North America, provide a system to examine the genomic mechanisms and consequences of invasion. While the novel range of *D. carinulata* has expanded beyond previous photoperiodic limits in < 15 generations, wide-spread hybridization of *D. carinata*, *D. elongata*, and *D. sublineata* has also been documented. I will present the development of genomic resources and ongoing research to understand how these two simultaneous processes may impact efficacy and safety of the biocontrol program. We developed high-quality chromosome-level reference genomes in collaboration with Ag100Pest, conducted genome-wide analysis of hybridization and range expansion from hundreds of individuals, and have begun to identify genetic variation associated with ecologically-relevant phenotypes. Our work can serve as a case-study for development and use of genomic tools to study arthropods in novel environments.<br><br>|
| <img src="/images/AGSX_speaker_Heath_Blackmon.png"/> __[Heath Blackmon](http://coleoguy.github.io/)__ <br><br>Texas A&M University, College Station, TX |<br>__Title:__ Realistic models reveal the causes of chromosome evolution<br><br>__Summary:__ The division of the genome into individual chromosomes is perhaps one of the simplest characteristics of a genome. Despite this simplicity, researchers have been largely unsuccessful in explaining why some clades display near stasis in chromosome number while others exhibit variation even among closely related species. We have developed a biologically realistic chromosome evolution model that allows us to test many long-standing verbal hypotheses. Results from applying this model in insects suggest that changes to chromosome number are deleterious and may often fix due to drift and that in some clades sexual antagonism may be a key driver of changes in chromosome number.<br><br>|
| <img src="/images/AGSX_speaker_Joshua_Benoit.png"/> __[Joshua B. Benoit](http://insectphysiology.uc.edu/)__ <br><br>University of Cincinnati, Cincinnati, OH |<br>__Title:__ Unique facets of sex and reproduction in insects identified through combined use of multiomic and functional studies<br><br>__Summary:__ Sexual aspects and reproduction of insect systems have significant variation that range from diverse mechanisms of pre-copulatory interactions to unique processes for allocation of nutrients to developing progeny. This seminar will feature three vignettes where combined proteomic, transcriptomic, and genomic methods followed by directed functional studies have been used to establish unique factors underlying copulation, viviparous reproduction, and external provisioning of nutrients to developing offspring. The insect systems discussed will be viviparous cockroaches and flies, the Antarctic midge, and a non-model fruit fly species.<br><br>|


<a name="apr13"></a>
**<span style="color:purple"> April 13 </span>** <br/>
Organized and moderated by Dr. Marcé D. Lorenzen, North Carolina State University, Raleigh, NC, USA


| __Session 3: Application of New Genomic Tools and Techniques in Arthropods__ | __[10-12:00 CST [US Central] ~ 11-13:00 EDT [US East Coast] ~ 17-19:00 CET [Europe]](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSX+2021+Session+3&iso=20210413T11&p1=263&ah=2)__ |
| ---- | ----- |
| <img src="/images/AGSX_speaker_Marcé_Lorenzen.png"/> __[Marcé Lorenzen](https://cals.ncsu.edu/entomology-and-plant-pathology/people/mdlorenz/)__ <br><br>Associate Professor, North Carolina State University, Raleigh, NC|<br>__Title:__  Developing Genomic Tools for Hemipterans<br><br>__Summary:__ Despite the power of CRISPR/Cas9-based genome editing, deploying this technology is a new species can be challenging. Here we discuss overcoming some of the hrudles we faced to generate the first gene edited corn planthopper, Peregrinus maidis, strains. While we had an excellent “guidebook” to follow – CRISPR/Cas9-based genome editing of the brown planthopper, Nilaparvata lugens – the process of deploying this technology in a new planthopper species was far from easy. Therefore to aid others working with hemipteran species, we plan to offer a number of tips and tricks we wish we had known. We will start from the first step, that of developing genomic and transcriptomic resources for P. maidis. The next steps involved developing protocols for harvesting and microinjecting precellular embryos, which are critical for Cas9-based genome editing, since DNA and proteins cannot easily cross cell membranes. The later steps cover Cas9-based genome editing itself, including selection of target genes. For example, drawing on the work performed in N. lugens, we initially targeted the pivotal eye-color gene, white (w), which results in white eyes when disrupted in the brown planthopper. However, while Cas9-mediated knockout of w was successful in generating P. maidis embryos that lacked eye-spot pigmentation, only embryos that still possessed eye-spot pigmentation hatched. Since this suggests that w may play a vital role in P. maidis development, we tested another eye-color gene, cinnabar (cn). Unlike w, loss-of-function mutations in cn had no apparent impact on development, allowing us to establish a red-eyed P. maidis colony. Importantly, these are merely the first steps towards a much boarder goal: bringing game-changing genomic tools to bear on understanding and controlling these important agricultural pests.<br><br>|
| <img src="/images/AGSX_speaker_Omar_Akbari.jpg"/> __[Omar Akbari](https://biology.ucsd.edu/research/faculty/oakbari)__ <br><br>Associate Professor, University of California San Diego, La Jolla, CA|<br>__Title:__  Engineering transgenic Drosophila suzukii for wild population suppression & eradication<br><br>__Summary:__ Throughout the world invasive species are increasingly affecting fisheries, food, agriculture and forestry resources. As a result of global trade, invasive species are continuously introduced into new environments where they become established and wreak havoc on agriculture, human health and the environment. The spotted wing fruit fly Drosophila suzukii originated in Japan and was first described in 1931. Since then, this fly has invaded most of Europe and America, posing significant burdens on more than 100 fruit crops. In this short talk, I will describe the molecular-genetic based technologies we are developing to combat this global invasive pest. <br><br>|
| <img src="/images/AGSX_speaker_Clay_Chu.png"/> __[Fu-Chyun (Clay) Chu](https://www.linkedin.com/in/fu-chyun-chu-57a08013b/)__ <br><br>Senior Scientist, All Things Bugs LLC, Midwest City, OK|<br>__Title:__  Two Marker System using CRISPR/Cas9 in Crickets and Mealworms<br><br>__Summary:__ The idea of insects as sustainable food ingredients has been the focus of a new emerging industry in recent years. As with any crop, insects can be engineered via gene editing and other means to provide more desirable phenotypes. Our research is focused on the two most commonly farmed edible insect species: house cricket (Acheta domesticus) and yellow mealworm (Tenebrio molitor). Our goal is to develop a genetic engineering system to increase the value of these insects as food crops, such as nutritional content, disease resistance or reduced allergenicity. In this work, we are using CRISPR/Cas9 technology to create both knock-in and knock-out phenotypes in both insect species through embryonic microinjection. To build this genetic engineering system, we tested two different marker systems in both species. The first marker is a recessive phenotype of the insect’s eye color. We selected the eye color related genes Ad vermillion (AdV) and Tm vermillion (TmV) in A. domesticus and T. molitor, respectively, as the sgRNA targeting sites, and were able to generate successful eye color knock-outs by CRISPR resulting in white eye color phenotype in these insects. The second marker used the CRISPR system to knock-in green fluorescent protein (EGFP) driven by the species-specific muscle actin promoters from A. domesticus and T. molitor. Overall, our results show 96% of the G0 A. domesticus crosses produced white eye offspring as well as several EGFP positive G1 offspring. Around 39% of the G0 T. molitor crosses provided white eyed G1 and G2 offspring and 16% of the G0 crosses produced EGFP positive G1 offspring. The eye color knock-out phenotype in T. molitor resulted in a white eye color in all life stages. However, since the cricket eye is pigmented by both ommochromes and pteridines, knocking out the vermillion gene in A. domesticus resulted in a red-eyed phenotype similar to vermilion knock out in flies. For the knock-in marker, EGFP expression was obvious in all muscles in all life stages in both species. By using this two marker system in A. domesticus and T. molitor, we demonstrate the potential to engineer insects for beneficial phenotypes.<br><br>|


<a name="may12"></a>
**<span style="color:purple"> May 12 </span>** <br/>
Organized and moderated by Drs. Sonia Eynard & Alain Vignal, INRAE, Castanet Tolosan, France


| __Session 4: Honeybee Workshop__ | __[10-12:00 CST [US Central] ~ 11-13:00 EDT [US East Coast] ~ 17-19:00 CET [Europe]](https://www.timeanddate.com/worldclock/fixedtime.html?msg=AGSX+2021+Session+4&iso=20210512T11&p1=263&ah=2)__ |
| ---- | ----- |
| <img src="/images/AGSX_speaker_Matthew_Webster.jpg"/> __[Matthew Webster](https://katalog.uu.se/profile/?id=N1-581)__ <br><br>Professor in the Department of Medical Biochemistry and Microbiology at Uppsala University (Sweden) |<br>__Title:__ Causes and consequences of extreme recombination rate in honeybees<br><br>__Summary:__ One of the most striking features of the genetics of honeybees is an exceedingly high rate of meiotic recombination, which is around 20 times higher than most other animals. Extreme recombination rates are also observed in other social insects but not in related solitary species. Although a number of theories have been proposed, the evolutionary explanation for these elevated recombination rates is unclear. High recombination rates also have important implications for molecular evolution and interpretation of genomic data in honeybees. Here I will discuss studies of the evolution of recombination in bees using both direct inference of crossover events in haploid drones and statistical inference based on patterns of linkage disequilibrium.<br><br>|
| <img src="/images/AGSX_speaker_Julia_Jones.png"/> __[Julia Jones](https://people.ucd.ie/julia.jones)__ <br><br>Assistant Professor in the School of Biology and Environmental Science at University College Dublin (Ireland) |<br>__Title:__ Investigating the evolution and maintenance of complex societies: from fanning bees to SNP chip development<br><br>__Summary:__ Effective division of labour among individuals in a honey bee colony is important for colony success, and genetic diversity is thought to be key to a colony’s ability to respond appropriately to the environment. Here I aim to highlight some of the different approaches we have used to help unravel the genomic mechanisms involved in the evolution and maintenance of sociality and division of labour.<br><br>|
| <img src="/images/AGSX_speaker_Amro_Zayed.png"/> __[Amro Zayed](https://biology.gradstudies.yorku.ca/faculty/a-zayed/)__ <br><br>Associate Professor in the Biology Department of the Center for Bee Ecology, Evolution and Conservation at York University (Canada) |<br>__Title:__ BeeCSI and bio-markers for bee health<br><br>__Summary:__ As York Research Chair in Genomics, Dr. Zayed leads a team of scientists that study the evolutionary, behavioural and conservation genomics of social insects, using the honey bee as a model organism.<br><br>|


<a name="mailing_list"></a> 
### **Mailing List**
To keep up to date with news about this conferece, the i5k webinar series and other i5k activities, please subscribe to the Arthropod News [mailing list](https://listserv.ksu.edu/?SUBED1=ARTHROPODNEWS&A=1). Email frequency is typically very low.

<a name="organizers"></a> 
### **Organizing Committee and Production Team**
This virtual symposium series was brought to you through the efforts of the following organizing committee and production team:
- [Brenda Oppert](https://www.ars.usda.gov/plains-area/mhk/cgahr/spieru/people/brenda-oppert/), USDA-ARS, Manhattan, KS, USA
- [Lindsey Perkin](https://www.ars.usda.gov/people-locations/person?person-id=53466), USDA-ARS, College Station, TX, USA
- [Marcé D. Lorenzen](https://cals.ncsu.edu/entomology-and-plant-pathology/people/mdlorenz/), North Carolina State University, Raleigh, NC, USA
- [Sonia Eynard](https://scholar.google.com/citations?user=CAloEFcAAAAJ&hl=en), INRAE, Castanet Tolosan, France
- [Alain Vignal](https://genphyse.toulouse.inra.fr/people/vignal), INRAE, Castanet Tolosan, France
- [Glenn Hanes](https://www.ars.usda.gov/people-locations/person?person-id=2280), USDA-ARS, Beltsville, MD, USA
- [Pia Olafson](https://www.ars.usda.gov/people-locations/person/?person-id=33760), USDA-ARS, Kerrville, TX, USA
- [Brad Coates](https://www.ars.usda.gov/people-locations/person?person-id=1047), USDA-ARS, Ames, IA, USA
